pkg/sql/sqlbase/structured.go

```diff
@@ -886,15 +885,21 @@ func generatedFamilyName(familyID FamilyID, columnNames []string) string {
 // in a similar way.
 func (desc *TableDescriptor) MaybeFillInDescriptor(
 	ctx context.Context, protoGetter protoGetter,
-) error {
+) (bool, error) {
```

Comment:
I don't think you want to change the signature of this function -- its return value is only used in one place, and that place is specifically meant for whether the FK representations changed. That function is a special case already, so I would be fine to not use MaybeFillDescriptor there.